extension/parquet/include/decode_utils.hpp

```diff
@@ -12,10 +12,12 @@ class ParquetDecodeUtils {
 	}
 
 	static const uint64_t BITPACK_MASKS[];
+	static const uint64_t BITPACK_MASKS_SIZE;
 	static const uint8_t BITPACK_DLEN;
 
 	template <typename T>
 	static uint32_t BitUnpack(ByteBuffer &buffer, uint8_t &bitpack_pos, T *dest, uint32_t count, uint8_t width) {
+		D_ASSERT(width < ParquetDecodeUtils::BITPACK_MASKS_SIZE);
```

Comment:
Can we turn this into an invalid input exception? The width is read from the file, so a corrupt file could cause an out of bounds read here.